index.html

```diff
@@ -80,16 +78,18 @@ <h3><b>CANOPY</b></h3>
                   <li>2015</li>
                 </ol>
               </div>
-              <p>
+              <p class="project-text">
                 A daily selection of privately personalized reads; no accounts
                 or sign-ups required.
               </p>
               <ul class="tag-list">
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
               </ul>
-              <button class="blocks-button">See Project</button>
+              <div class="blocks-buttonWrapper">
+                <button class="blocks-button">See Project</button>
+              </div>
```

Comment:
- [ ] Same as this issue, please make sure you fix it, compare the 2 images, and bring all the necessary changes to your design.

<img width="807" alt="1" src="https://user-images.githubusercontent.com/77843050/227736457-cde54d9c-d055-43a2-b4cb-86b6b885b446.png">